Keystrokes to adjust applications volume and mute [take 2] (#16591)
nvaccess/nvda: user_docs/en/userGuide.md

Note that it is necessary to check at least one mode.
This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.

##### Volume of other applications {#OtherAppVolume}

This slider allows you to adjust the volume of all currently running applications other than NVDA.
This volume setting will apply to all other applications sound output, even if they start after this setting is changed.
This volume can also be controlled via the following keyboard commands from anywhere:

| Name | Key | Description |
|---|---|---|
| Increase applications volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
| Decrease applications volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |

This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.

> ##### Applications volume adjuster status {#AppsVolumeAdjusterStatus}

Reviewer: This paragraph is not clear to me. The user may not understand what the application volume adjuster is. Also the term "adjuster" is not common; "other apps volume control" would be more common, or even just "other apps volume".
But the problem seems broader.

Note that it is necessary to check at least one mode.
This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.

##### Volume of other applications {#OtherAppVolume}

This slider allows you to adjust the volume of all currently running applications other than NVDA.
This volume setting will apply to all other applications sound output, even if they start after this setting is changed.
This volume can also be controlled via the following keyboard commands from anywhere:

| Name | Key | Description |
|---|---|---|
| Increase applications volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
| Decrease applications volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |

This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.

> ##### Applications volume adjuster status {#AppsVolumeAdjusterStatus}

Reviewer 2: @CyrilleB79 I see there is a sentence at the top now, is that updated since this comment?

Also re modes, I see mention of modes in the code, but not (now?) in the User Guide.  While the terminology should ideally be consistent, the presentation to the user in this case should be - I see the previous notes on Sound Split talk about modes of THAT function.

> ##### Applications volume adjuster status {#AppsVolumeAdjusterStatus}

Reviewer: > @CyrilleB79 I see there is a sentence at the top now, is that updated since this comment?

Yes, @mltony has added this sentence after my comment.

My opinion:
The information in this paragraph is not very clear, but that's not a matter of documentation. That's the best we can do with the current UX design. Only an alternative design such as the proposal in https://github.com/nvaccess/nvda/pull/16591#issuecomment-2255582948 could clarify more.

If you do not accept https://github.com/nvaccess/nvda/pull/16591#issuecomment-2255582948 the documentation is OK.

> Also re modes, I see mention of modes in the code, but not (now?) in the User Guide. While the terminology should ideally be consistent, the presentation to the user in this case should be - I see the previous notes on Sound Split talk about modes of THAT function.

I do not understand what you mean. It's worth noting that "mode" was appearing either in the UG or in the GUI for the newly introduced features; but @mltony has modified the wording to answer my comment on this topic.

Regarding the existing features, e.g. sound split, changing the wording is off-topic of this PR in any case. And I wonder if it may even be desirable at all considering that people are already used to the current wording.

Note that it is necessary to check at least one mode.
This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.

##### Applications volume adjuster status {#AppsVolumeAdjusterStatus}

This combo box allows you to select the status of the applications volume adjuster.

> Applications volume adjuster allows you to adjust volume of all other applications except for NVDA or mute them with a single keystroke.

Reviewer 3: ```suggestion
The applications volume adjuster allows you to adjust volume of all other applications except for NVDA or mute them with a single keystroke.
```

Possible values are:

* Disabled: NVDA doesn't interfere with volume levels of other applications.
* Enabled: volume of other applications can be adjusted via [other applications volume slider](#OtherAppVolume).
* Muted: all other applications are muted.

The following keyboard command can also be used from anywhere:

| Name | Key | Description |
|---|---|---|

> | Toggle Applications volume adjuster status | `NVDA+alt+delete` | Cycles through options of Applications volume adjuster status. |
> If this option is set to Muted, then its value will not be stored and it will be restored to Enabled upon next restart of NVDA.

Reviewer 3: ```suggestion
| Toggle "Applications volume adjuster" status | `NVDA+alt+delete` | Cycles through statuses for "Applications volume adjuster" |

If this option is set to muted, then its value will not be stored and it will be restored to enabled upon next restart of NVDA.
```

This check box allows you to mute or unmute all applications except NVDA at once.
This mute setting will apply to all other applications sound output, even if they start after this setting is changed.

The following keyboard command can also be used from anywhere:

| Name | Key | Description |
|---|---|---|

> | Mute or unmute other applications | `NVDA+alt+delete` | Cycles through statuses for "Applications volume adjuster" |

Reviewer 3: ```suggestion
| Mute or unmute other applications | `NVDA+alt+delete` | Toggles mute/unmute on other applications |
```
